app/components/user_menu/signed_in_component.rb

```diff
@@ -24,4 +24,16 @@ def conversations?
   def admin?
     user.admin?
   end
+
+  def unread_notifications
```

Comment:
The methods return an array rather than a boolean... should they still be questions? 

I could also simplify things by making them actually return a boolean - I think the reason they aren't is that I was trying to plan ahead a little bit as we had talked about making new notifications be a dropdown from the notification bell in the future.

Reply:
Yeah, let's keep it as a question so it moves all the logic, `#any?`, to the Ruby code instead of the view.

Reply:
Yes, this was corrected in a recent commit. Marking this convo as resolved.